docs/Tools/buildScriptsDocs.py

```diff
@@ -14,6 +14,8 @@
 import subprocess
 import shlex
 
+SUPER_DEBUG = False
```

Comment:
Maybe just a comment to explain this global variable ?

Reply:
Done